### PR TITLE
[ADD] base: add Arabic languge

### PR DESCRIPTION
**Pull request by mart-e**
Odoo only has `ar_SY` language but not a generic Arabic language.

cc @odony to be sure it is a stable change

### CODE REVIEW

**Comment by sswapnesh** [comment]
May be better to add in master this csv will be loaded when someone upgrade `base` module.

**Comment by mart-e** [comment]
Well as odoo finally works well with Arabic languages, it is the right time to have a proper Arabic language.

**Comment by mart-e** [comment]
robodoo r+

File: odoo/addons/base/data/res.lang.csv [hunk 1]
```diff
@@ -1,8 +1,10 @@
 "id","name","code","iso_code","direction","grouping","decimal_point","thousands_sep","date_format","time_format","translatable","week_start"
 "base.lang_en","English","en_US","en","Left-to-Right","[3,0]",".",",","%m/%d/%Y","%H:%M:%S","True","7"
 "base.lang_am_ET","Amharic / አምሃርኛ","am_ET","am_ET","Left-to-Right","[3,0]",".",",","%d/%m/%Y","%I:%M:%S","True","7"
-"base.lang_ar_SY","Arabic / الْعَرَبيّة","ar_SY","ar","Right-to-Left","[3,0]",".",",","%d %b, %Y","%I:%M:%S","True","6"
```
Comment (odony) [inline odoo/addons/base/data/res.lang.csv]: Let's use `ar_AA` (Arabic "Unitag") as a countryless ISO code for arabic :-)

**Comment by mart-e** [comment]
robodoo r+

**Comment by KangOl** [comment]
This patch wasn't stable due to uniqness of lang name.
https://github.com/odoo/odoo/blob/491deeb3d4227557b15e31b4a8e93022ebb4ff90/odoo/addons/base/models/res_lang.py#L52

**Pull request closed without merging**